tests/cxx/iwyu_stricter_than_cpp.cc

```diff
@@ -7,6 +7,8 @@
 //
 //===----------------------------------------------------------------------===//
 
+// ARGS: -Xiwyu --check_also="tests/cxx/*-autocast.h" -Xiwyu --check_also="tests/cxx/*-fnreturn.h" -Xiwyu --check_also="tests/cxx/*-typedefs.h" -Xiwyu --check_also="tests/cxx/*-d2.h" -I .
```

Comment:
It might be a good idea to support multi-line ARGS strings to avoid very long lines like this, but that feature seems to be a bit tricky to implement.

Reply:
It should be possible to do something with a trailing `\` to indicate line continuation... Once the tactical changes I suggested are applied, I can experiment with that on top of your base.